draft-ietf-jsonpath-base.md

```diff
@@ -1408,15 +1408,15 @@ is a subset of the set of instances of the other type.
 {{tbl-types}} defines the available types in terms of abstract instances, where `n` denotes a node, `v` denotes a value, and `nl` denotes
 a non-empty nodelist. The table also lists the subtypes of each type.
 
-| Type                  | Abstract Instances                       | Subtypes                                 |
-| :--:                  | :----------------:                       | :------:                                 |
-| `OptionalNodeOrValue` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNode`, `OptionalValue`, `Value` |
-| `OptionalNode`        | `Node(n)`, `Nothing`                     |                                          |
-| `OptionalValue`       | `Value(v)`, `Nothing`                    | `Value`, `OptionalBoolean`               |
-| `Value`               | `Value(v)`                               | `Boolean`                                |
-| `OptionalBoolean`     | `Value(true)`, `Value(false)`, `Nothing` | `Boolean`                                |
-| `Boolean`             | `Value(true)`, `Value(false)`            |                                          |
-| `OptionalNodes`       | `Nodes(nl)`, `Nothing`                   | `OptionalNode`                           |
+| Type                  | Abstract Instances                       | Subtypes                                                               |
+| :--:                  | :----------------:                       | :------:                                                               |
```

Comment:
BTW, is there a reason you want those centered?

Reply:
No, it would be fine for them to be left-justified.

Reply:
```suggestion
| :--                  | :----------------                       | :------                                                               |
```